adapter_pipelines/submit.wdl

```diff
@@ -14,14 +15,28 @@ task get_metadata {
     hash_start = url.rfind('/', 0, hash_end) + 1
     hash = url[hash_start:hash_end]
     print(hash)
+    CODE
 
+    python <<CODE > bearer_token.txt
+    if "${use_caas}" == "true":
+        from oauth2client.service_account import ServiceAccountCredentials
+        json_credentials="/cromwell-metadata/caas_key.json"
+        scopes = ['https://www.googleapis.com/auth/userinfo.profile', 'https://www.googleapis.com/auth/userinfo.email']
+        credentials = ServiceAccountCredentials.from_json_keyfile_name(json_credentials, scopes=scopes)
+        print(credentials)
     CODE
 
     # Get metadata from Cromwell for this workflow id
-    creds=/cromwell-metadata/cromwell_credentials.txt
-    curl -u $(cut -f1 $creds):$(cut -f2 $creds) \
-      --compressed \
-      "https://cromwell.mint-${runtime_environment}.broadinstitute.org/api/workflows/v1/$(cat workflow_id.txt)/metadata?expandSubWorkflows=true" > metadata.json
+    if [ ${use_caas} = true ]; then
+        echo $(cat bearer_token.txt)
```

Comment:
Why `echo` here?

Reply:
Oh I left this in from debugging! Thanks for catching that!